amads/melody/contour/interpolation_contour.py

```diff
@@ -0,0 +1,272 @@
+"""
+Calculates the Interpolation Contour of a melody, along with related features, as
+implemented in the FANTASTIC toolbox of Müllensiefen (2009).
```

Comment:
Does Müllensiefen provide any original citation for this?

Reply:
fixed